package.json

```diff
@@ -133,7 +133,7 @@
     "@percy/cypress": "1.0.5",
     "@vue/test-utils": "1.0.0-beta.25",
     "autoprefixer": "9.4.2",
-    "axe-cli": "3.0.0",
+    "axe-core": "3.2.2",
```

Comment:
We weren't using `axe-cli` and in fact require the underlying `axe-core` in our code, so this makes that dependency explicit. There is an added benefit that this no longer installs `chromedriver` in the background.